install/py2-warning.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if [ ! -f 'install.sh' ]; then echo 'Where are you?'; exit 1; fi
+
+source ./install/docker-aliases.sh
+
+py2_warning() {
+    $dcr --no-deps --entrypoint python web --version | grep -q 'Python 2'
+    if [[ $? -eq 0 ]]; then
```

Comment:
I'm unable to see the failure case:

```diff
diff --git a/install/py2-warning.sh b/install/py2-warning.sh
index b6462bb..bea0133 100644
--- a/install/py2-warning.sh
+++ b/install/py2-warning.sh
@@ -5,7 +5,7 @@ if [ ! -f 'install.sh' ]; then echo 'Where are you?'; exit 1; fi
 source ./install/docker-aliases.sh
 
 py2_warning() {
-    $dcr --no-deps --entrypoint python web --version | grep -q 'Python 2'
+    $dcr --no-deps --entrypoint python web --version | grep -q 'Python 9'
     if [[ $? -eq 0 ]]; then
     cat <<"EOW"
  _  _   ____      ____  _       _______     ____  _____  _____  ____  _____   ______   _  _

```

```
$ bash install/py2-warning.sh 
Creating sentry_onpremise_web_run ... 
Creating sentry_onpremise_web_run ... done

$
```

Reply:
I'll check this a few more times, CI keeps failing too.

Reply:
Oh wait, this is expected as you are using Python 3. This does not check for Python 3 but checks for Python 2. So it _only_ warns if the output matches your `grep`. If you used `Python 3` it would have warned you ;)

Reply:
Ah yes, of course.

```
$ bash install/py2-warning.sh
Creating sentry_onpremise_web_run ... 
Creating sentry_onpremise_web_run ... done
 _  _   ____      ____  _       _______     ____  _____  _____  ____  _____   ______   _  _
| || | |_  _|    |_  _|/ \     |_   __ \   |_   \|_   _||_   _||_   \|_   _|.' ___  | | || |
| || |   \ \  /\  / / / _ \      | |__) |    |   \ | |    | |    |   \ | | / .'   \_| | || |
| || |    \ \/  \/ / / ___ \     |  __ /     | |\ \| |    | |    | |\ \| | | |   ____ | || |
|_||_|     \  /\  /_/ /   \ \_  _| |  \ \_  _| |_\   |_  _| |_  _| |_\   |_\ `.___]  ||_||_|
(_)(_)      \/  \/|____| |____||____| |___||_____|\____||_____||_____|\____|`._____.' (_)(_)

You are using Sentry with Python 2, which is deprecated.
Sentry 21.1 will be the last version with Python 2 support.

$ git diff
```

```diff
diff --git a/install/py2-warning.sh b/install/py2-warning.sh
index b6462bb..b52ae82 100644
--- a/install/py2-warning.sh
+++ b/install/py2-warning.sh
@@ -5,7 +5,7 @@ if [ ! -f 'install.sh' ]; then echo 'Where are you?'; exit 1; fi
 source ./install/docker-aliases.sh
 
 py2_warning() {
-    $dcr --no-deps --entrypoint python web --version | grep -q 'Python 2'
+    $dcr --no-deps --entrypoint python web --version | grep -q 'Python 3'
     if [[ $? -eq 0 ]]; then
     cat <<"EOW"
  _  _   ____      ____  _       _______     ____  _____  _____  ____  _____   ______   _  _

$
```

 There it is. :)